File: rss-reader/app/src/main/java/com/abubusoft/kripton/examples/rssreader/service/persistence/RssDataSource.java
package com.abubusoft.kripton.examples.rssreader.service.persistence;

import com.abubusoft.kripton.android.annotation.BindDataSource;

@BindDataSource(fileName = "rss.db",daoSet = {DaoRss.class, DaoArticle.class, DaoChannel.class}, asyncTask = true)
public interface RssDataSource {
}

File: kotlin-rss-reader/app/src/main/java/com/abubusoft/kripton/example/rssreader/service/persistence/RssDataSource.kt
package com.abubusoft.kripton.example.rssreader.service.persistence

import com.abubusoft.kripton.android.annotation.BindDataSource


@BindDataSource(fileName = "rss.db", daoSet = [DaoRss::class, DaoArticle::class, DaoChannel::class], asyncTask = true)
interface RssDataSource
